tests/modules/test_math.py

```diff
@@ -76,8 +76,8 @@ def test_constants(self):
         self.assertCodeExecution("""
             import math
             print(math.e)
-            print(math.inf)
-            print(math.nan)
+            # print(math.inf) # Not until CPython 3.5
```

Comment:
Is there any reason not to do the simple `if sys.version() >= (3,5)` trick here?